Dockerfile.svc

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7-alpine
+FROM python:3.7-alpine as base
```

Comment:
You don't use `base` anywhere, is this needed?

Reply:
no, I was trying a few different things and this was left over, thanks!